<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyAddForm</class>
 <widget class="QWidget" name="keyAddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增密钥</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密钥类型:                          密钥数量:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKeyType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbKeyCount">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-- by renrendoushikexuejia</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
